content/docs/user-guide/external-dependencies.md

```diff
@@ -41,24 +41,6 @@ stage to your list of stages in dvc.yaml.
 > Note that some of these commands use the `/home/shared` directory, typical in
 > Linux distributions.
 
-### Local file system path
-
-```dvc
-$ dvc run -n download_file
-          -d /home/shared/data.txt \
-          -o data.txt \
-          cp /home/shared/data.txt data.txt
-```
-
-### SSH
-
-```dvc
-$ dvc run -n download_file
-          -d ssh://user@example.com:/home/shared/data.txt \
-          -o data.txt \
-          scp user@example.com:/home/shared/data.txt data.txt
-```
-
```

Comment:
Sorry, slightly unrelated: reordered these 2 sections to match the order in remote add/modify.

Reply:
Same in content/docs/user-guide/external-dependencies.md